.github/workflows/create.yml

```diff
@@ -48,8 +58,19 @@ jobs:
           architecture: ${{ matrix.architecture }}
 
       - name: Build wheels
+        if: ${{ matrix.cibw_archs != 'aarch64' }}
+        env:
+          CIBW_SKIP: '*musllinux* ${{ matrix.cibw_skip }}'
+        run: python -m cibuildwheel --output-dir dist
+
+      - name: Build and test aarch64 wheels
+        if: ${{ matrix.cibw_archs == 'aarch64' }}
         env:
           CIBW_SKIP: '*musllinux* ${{ matrix.cibw_skip }}'
+          CIBW_ARCHS: 'aarch64'
+          CIBW_TEST_REQUIRES: 'pytest'
+          CIBW_BEFORE_TEST: 'yum install ant -y && cd {project} && ant all'
+          CIBW_TEST_COMMAND: 'cd {project}/tests/ && CLASSPATH=../build/test-classes:../build/classes python -m pytest -v'
```

Comment:
Optimization: we can expand the tests also to the `x86_64` version, so there's no need to keep a separate step for `aarch64` wheels.